src-built-in/components/UIAPI/reducers/linker.spec.ts

```diff
@@ -5,162 +5,35 @@
  * directory for now (personal pref) I can move them
  * to another folder later on if requested to do so.
  */
-import { linker as reducer, initialState } from './linker';
-import { loop, Cmd } from 'redux-loop';
+import { linker as reducer , initialState } from './linker';
 import { linkChannel, initializeLinker, cleanUp, fitDOM } from '../effects/linker';
 import { toggleSuccess, toggleFailure, initSuccess } from '../actions/linkerActions';
-import * as actions from '../actions/linkerActions';
+import { actions } from '../types'
 import { assert } from 'chai';
 import 'mocha';
 
 /**
  * Creates a dummy channel object for testing purposes
  */
-const channel = (id: number, active: boolean, color: string) => {
+const channel = (id: string, active: boolean, color: string) => {
     return {
         name: color,
         color,
         active,
-        id
+        id,
+        border: color
     }
 }
 
```

Comment:
Are we going to test the linker's reducer anymore?

Reply:
Yes ma'am - just not for this PR. It's on the list to go back around. Because it changed so much and we want to get the finsemble-UI up and running, I think it's worth skipping. I will create a card to get the tests rolling.